redash/destinations/discord.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import requests
+
+from redash.destinations import BaseDestination, register
+from redash.utils import json_dumps
+
+RED_ALERT_COLOR = "12597547"
+
+GREEN_ALERT_COLOR = "2600544"
+
+
+class Discord(BaseDestination):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {"type": "string", "title": "Discord Webhook URL"}
+            },
+            "secret": ["url"],
+            "required": ["url"],
+        }
+
+    @classmethod
+    def icon(cls):
+        return "fa-discord"
+
+    def notify(self, alert, query, user, new_state, app, host, options):
+        # Documentation: https://birdie0.github.io/discord-webhooks-guide/discord_webhook.html
+        fields = [
+            {
+                "name": "Query",
+                "value": "{host}/queries/{query_id}".format(
+                    host=host, query_id=query.id
+                ),
+                "inline": True,
+            },
+            {
+                "name": "Alert",
+                "value": "{host}/alerts/{alert_id}".format(
+                    host=host, alert_id=alert.id
+                ),
+                "inline": True,
+            },
+        ]
+        if alert.options.get("custom_body"):
+            fields.append({"name": "Description", "value": alert.options["custom_body"]})
+        if new_state == alert.TRIGGERED_STATE:
+            if alert.options.get("custom_subject"):
+                text = alert.options["custom_subject"]
+            else:
+                text = f"{alert.name} just triggered"
+            color = RED_ALERT_COLOR
```

Comment:
```suggestion
            color = colors.get(new_state, "red")
```

Reply:
Same situation as with decimal codes in constants, fixed